### PR TITLE
Add eks-helm modules - Metrics-server

### PR DESCRIPTION
**Pull request by spartan-tainguyen**
## Summary
Add a new module: eks-helm/metrics-server to provision and configure metrics server in eks.

## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [ ] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

### CODE REVIEW

File: aws/eks-helm/metrics-server/variables.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  default     = "kube-system"
+}
+variable "helm_release_name" {
+  description = "The Helm release of the services."
+  type        = string
+  default     = "metrics-server"
+}
+
+variable "helm_chart_version" {
+  default     = "3.11.0"
```
Comment (spartan-toannhan) [inline aws/eks-helm/metrics-server/variables.tf:13]: use the latest version https://artifacthub.io/packages/helm/metrics-server/metrics-server

Comment (spartan-tainguyen) [reply aws/eks-helm/metrics-server/variables.tf]: updated

File: aws/eks-helm/metrics-server/examples/main.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module "example" {
+  source = "../"
+
+  helm_release_name  = "metrics-server"
+  namespace          = "kube-system"
+  helm_chart_version = "3.12"
```
Comment (spartan-toannhan) [inline aws/eks-helm/metrics-server/examples/main.tf:6]: should `3.12.2` ?

Comment (spartan-tainguyen) [reply aws/eks-helm/metrics-server/examples/main.tf]: I forgot the example, I updated it, thank you.

**Review by spartan-toannhan — APPROVED** [review]

**Pull request merged**